plasmapy/physics/__init__.py

```diff
@@ -36,6 +36,4 @@
 
 from .relativity import Lorentz_factor
 
-from plasmapy import transport
-
```

Comment:
... wow, I have no idea why this was here. I'm surprised things didn't break by removing this!